login-workflow/src/new-architecture/screens/AccountDetailsScreen/AccountDetailsScreen.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import { AccountDetailsScreenBase, AccountDetailsScreenProps } from '../AccountDetailsScreen';
+import { useLanguageLocale } from '../../hooks';
+import { useRegistrationWorkflowContext } from '../../contexts';
+
+type AccountDetailsFullScreenProps = AccountDetailsScreenProps & {
+    title?: string;
+    instructions?: string;
+};
+
+export const AccountDetailsScreen: React.FC<AccountDetailsFullScreenProps> = (props) => {
+    const { t } = useLanguageLocale();
+    const regWorkflow = useRegistrationWorkflowContext();
+    const { nextScreen, previousScreen, screenData } = regWorkflow;
+    const [firstName, setFirstName] = useState(screenData.AccountDetails.firstName ?? '');
+    const [lastName, setLastName] = useState(screenData.AccountDetails.lastName ?? '');
+
+    const {
+        title = t('bluiRegistration:REGISTRATION.STEPS.ACCOUNT_DETAILS'),
+        instructions = t('bluiRegistration:REGISTRATION.INSTRUCTIONS.ACCOUNT_DETAILS'),
+        firstNameLabel = t('bluiCommon:FORMS.FIRST_NAME'),
+        /* eslint-disable @typescript-eslint/no-unused-vars */
+        initialFirstName,
+        initialLastName,
```

Comment:
You can remove these prop variables from here right? Anyway you are using these props below in the code.